custom_components/tuya_local/devices/point_panel_heater.yaml

```diff
@@ -0,0 +1,51 @@
+name: Point Panel Heater
+products:
+  - id: nqj51hxb2tx5kklv
+    name: Point Panel Heater POPANW600
+primary_entity:
+  entity: climate
+  translation_only_key: heater
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - dps_val: true
+          value: "heat"
+        - dps_val: false
+          value: "off"
+    - id: 2
+      name: temperature
+      type: integer
+      unit: C
+      mapping:
+        - scale: 0.1
+      range:
+        min: 5
+        max: 50
+    - id: 3
+      name: current_temperature
+      type: integer
+    - id: 4
+      name: preset_mode
+      type: string
+      optional: true
+      mapping:
+        - dps_val: ECO
+          value: eco
+        - dps_val: Comfort
+          value: comfort
+secondary_entities:
+  - entity: lock
+    translation_key: child_lock
+    category: config
+    dps:
+      - id: 7
+        type: boolean
+        name: lock
+  - entity: switch
+    translation_key: display_auto
```

Comment:
There is currently no translation key for "display_auto". What does this switch do? Perhaps it is better as a light with translation key `display`?